docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "2.1"
+services:
+  db:
+    image: mysql/mysql-server:5.7
+    environment:
+      - MYSQL_DATABASE=trademotors
+      - MYSQL_USER=trademotors
+      - MYSQL_PASSWORD=sitture2
+      - MYSQL_ROOT_HOST=0.0.0.0
```

Comment:
how about if we store the shared variables in a `.env` file and reference in the file. E.g. https://docs.docker.com/compose/environment-variables/#setting-environment-variables-in-containers

Reply:
Sounds Good!